### ISSUE TITLE
Does not work on Thunderbird 60

### ISSUE THREAD

**Issue by Giermann** [Giermann/WebAppTabs#1]
- does not display any buttons
- does not open clicked links

I assume the content policies do work different...

**Comment by Giermann** [comment]
@geekley:
Are you still working on this addon, or at least do you have more insight than me?

**Comment by geekley** [comment]
No, I only fixed it for my own use, and decided to share the code.
It's working in 52.9.1. Does it stop working when upgrading to 60? Then I guess I'll just avoid upgrading for now...

I don't know the code in-depth, and I don't intend to support the addon either (that's why I archived the repo). But if you find anything, please let me know!

**Comment by Giermann** [comment]
Yes, it stops working on Thunderbird 60. I decided to avoid the update 3 months ago, but now received it via auto-update.
I started the first conversions of now unsupported deprecated code and did a lot of debugging, but still don't know why it is not handling the content as it did before.

I'll keep you updated, in case I find something to fix it...

**Comment by geekley** [comment]
Well, from what I rememmber, I didn't actually use the code on github when I fixed the icon, I used the code from the zipped .xpi extension, then I changed it and re-zipped it to put it in downloads section. Then I made those changes again for github. I think the code in github wasn't reflecting the code in the extension, or I picked the wrong commit or something.

I suggest you try making those changes for the code in the xpi instead of cloning the repo.
And, if you want to go to the trouble of figuring out why the xpi is different from the repo, or what was the right commit to base the fork on, then you would be able to fix the repo as well.

**Comment by geekley** [comment]
@Giermann In case you're interested, I [fixed it for Thunderbird 60](https://github.com/geekley/WebAppTabs/releases/tag/release_4.0.60). There's still some issues (the tabs sometimes won't close or change URL) but at least it works for now. That is, until they update Thunderbird again and break the whole thing :man_shrugging:.
I think porting it beyond v60 isn't worth the trouble because it would require a complete rewrite, it seems.
